includes/AcquiaLiftAPIV2.inc

```diff
@@ -470,7 +470,7 @@ class AcquiaLiftAPIV2 extends AcquiaLiftAPIBase implements AcquiaLiftLearnReport
     $message = $this->canonicalizeRequest($verb, $host, $path, $query_args, $authorization, $timestamp, $content_type, $payload_hash);
 
     // Always use binary output
-    $signature = base64_encode(hash_hmac('sha256', $this->api_private_key, (string) $message, TRUE));
+    $signature = base64_encode(hash_hmac('sha256', (string) $message, $this->api_private_key, TRUE));
```

Comment:
That seems to be correct. The only doubt I have is the (string), would there ever be a case where the message is not a string but a boolean (FALSE for example)? If so, then the (string) should stay but maybe we need to add a comment above as to why this is. I think, ages ago, there was a bug where it required you to convert a boolean to a string as the hash that then got returned, without the conversion, was different.